src/components/dev-hub/series.js

```diff
@@ -34,8 +34,8 @@ const activeLiStyles = css`
     font-size: 22px;
     &:after {
         /* Adjust the background bullet position for the bull's eye text */
-        font-size: 38px;
-        top: -2px;
+        font-size: 54px;
+        top: 1px;
```

Comment:
I think the active bullet needs to drop about 1-2px lower to be even with the text